packages/forms-web-app/src/views/full-appeal/submit-final-comment/final-comment-submitted.njk

```diff
@@ -0,0 +1,31 @@
+{# todo: change layout based on appeal type when component reused #}
+{% extends "layouts/full-appeal-banner/main.njk" %}
+
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% block pageTitle %}
+  Final comment submitted - Appeal a planning decision - GOV.UK
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+			{{ govukPanel({
+        titleText: "Final comment submitted",
+				html: "Your reference number <br /><strong>0123456</strong>"
```

Comment:
does the reference number need to be a variable passed in from controller?

Reply:
done